src/components/policy/policy_external/test/sql_pt_representation_test.cc

```diff
@@ -83,9 +83,12 @@ class SQLPTRepresentationTest : public SQLPTRepresentation,
   // Gtest can show message that this object doesn't destroyed
   static std::auto_ptr<policy_handler_test::MockPolicySettings>
       policy_settings_;
+  static const std::string kAppStorageFolder;
 
   static void SetUpTestCase() {
-    const std::string kAppStorageFolder = "storage_SQLPTRepresentationTest";
+    if (!in_memory_) {
```

Comment:
@ZhdanovP propose not to use conditioned unit tests. Unit tests that checks policy functionality should be only in RAM. There should be another file with couple of basic unit tests that checks that RAM is also work